@here/harp-datasource-protocol/lib/Techniques.ts

```diff
@@ -1089,8 +962,166 @@ export interface TextTechnique extends BaseTechnique {
      * Fading time for labels in seconds.
      */
     textFadeTime?: number;
+    /**
+     * Clip the line outside the tile if `true`.
+     */
+    clipping?: boolean;
+    /**
+     * The name of the property containing the text.
+     */
+    textLabel?: string;
+}
+
+/**
+ * Runtime representation of [[SquaresStyle]] as parsed by [[StyleSetEvaluator]].
+ */
+export interface SquaresTechnique extends PointTechniqueParams {
```

Comment:
Maybe a good time to split this huge file into two, one for the params and one for the actual techniques

Reply:
Done.